e2e/migrations

```diff
@@ -0,0 +1 @@
+../migrations/
```

Comment:
This symlink is unnecessary. We can change the cwd directory before we execute `authgear` or `authgear-portal`.

Reply:
Currently `authgear` and `authgear-portal` rely uses env file from cwd though. Ok to update the their cmd logic for this case?